### PR TITLE
Signed Callbacks in Common Guide

### PR DESCRIPTION
**Pull request by vshisterov**
## Description

Since Voice API (will shortly) support Signed Webhooks by default, in the same form as Messages and Dispatch APIs do, 
* the *Signed Webhooks* guide moved from Messages API to common Webhooks guide,
* links in Messages *Configure Webhooks* and Voice *Webhook Reference* updated,
* Voice *Signed Callbacks [Beta]* guide removed.

### CODE REVIEW

File: _documentation/en/voice/voice-api/webhook-reference.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Vonage uses webhooks alongside its Voice API to enable your application to inter
 
 For more general information, check out our [webhooks guide](/concepts/guides/webhooks).
 
+## Signed Webhooks
+
+Signed webhooks are a way to verify that the request is coming from Vonage and its payload has not been tampered with during transit. Voice API, as well as [Messages](/messages/overview) and [Dispatch](/dispatch/overview) APIs, supports signed callbacks by default. See [Decoding signed webhooks](/concepts/guides/webhooks#decoding-signed-webhooks) to learn how to decode an incoming JWT signature.
```
Comment (abdulajet) [inline _documentation/en/voice/voice-api/webhook-reference.md:20]:
```suggestion
Signed webhooks are a way to verify that the request is coming from Vonage and its payload has not been tampered with during transit. Voice API, as well as [Messages](/messages/overview) and [Dispatch](/dispatch/overview) APIs, support signed callbacks by default. See [Decoding signed webhooks](/concepts/guides/webhooks#decoding-signed-webhooks) to learn how to decode an incoming JWT signature.
```

File: _documentation/en/concepts/guides/webhooks.md [hunk 2]
```diff
@@ -60,14 +60,81 @@ Information about your request is then sent to your webhook endpoint.
...
 
-Signed webhooks are a way to verify that the request is coming from Vonage and its payload has not been tampered with during transit. The [signed webhooks guide](/messages/concepts/signed-webhooks) has more details on its implementation and usage.
+Signed webhooks are supported by Messages, Dispatch and Voice APIs and are enabled by default. They provide a method for your application to verify a request is coming from Vonage and its payload has not been tampered with during transit. When receiving a request, the incoming webhook will include a JWT token in the authorization header which is signed with your signature secret.
 
-The following code snippets show how to decode an incoming JWT signature in a variety of languages:
+Validating signed webhooks provides a number of security benefits, including:
+
+* Ability to verify a request originates from Vonage
+* Ensures that the message has not been tampered with while in transit
+* Defends against interception and later replay
```
Comment (abdulajet) [inline _documentation/en/concepts/guides/webhooks.md:69]:
```suggestion
* The ability to verify a request originates from Vonage
* Ensuring that the message has not been tampered with while in transit
* Defending against interception and later replay
```

File: _documentation/en/concepts/guides/webhooks.md [hunk 3]
```diff
@@ -60,14 +60,81 @@ Information about your request is then sent to your webhook endpoint.
...
+
+### Validating signed webhooks
+
+There are two parts to validating signed webhooks:
+
+1. Verify the request
+2. Verify the payload (optional)
```
Comment (abdulajet) [inline _documentation/en/concepts/guides/webhooks.md:76]:
```suggestion
1. Verifying the request
2. Verifying the payload (optional)
```

File: _documentation/en/concepts/guides/webhooks.md [hunk 4]
```diff
@@ -60,14 +60,81 @@ Information about your request is then sent to your webhook endpoint.
...
+
+#### Verify the request
```
Comment (abdulajet) [inline _documentation/en/concepts/guides/webhooks.md:78]:
```suggestion
#### Verifying the request
```

File: _documentation/en/concepts/guides/webhooks.md [hunk 5]
```diff
@@ -60,14 +60,81 @@ Information about your request is then sent to your webhook endpoint.
...
+`alg` | HS256
+`typ` | JWT
+
+#### Signed JWT Payload
+
+The contents of the signed JWT payload are described in the following table, using the values included in the sample signed JWT shown previously:
+
+Field | Example Value | Description
+--- | --- | ---
+`iat` |  `1587494962` | The time at which the JWT was issued. Unix timestamp in SECONDS.
```
Comment (abdulajet) [inline _documentation/en/concepts/guides/webhooks.md:130]: any reason for seconds to be in caps?

**Review by abdulajet — APPROVED** [review]

File: _documentation/en/concepts/guides/webhooks.md (hunk 5 above)
Comment (vshisterov) [reply _documentation/en/concepts/guides/webhooks.md:136]: hm not sure... it was like this in the old guide.

**Pull request merged**